app/services/claims/fee_calculator/calculate.rb

```diff
@@ -0,0 +1,161 @@
+require 'ostruct'
```

Comment:
Is this `require` needed given we seem to be using the `Struct` [class](https://ruby-doc.org/core-2.5.0/Struct.html) instead of the [OpenStruct one](https://ruby-doc.org/stdlib-2.5.1/libdoc/ostruct/rdoc/OpenStruct.html) ?